docs/manual/config/cyclonedds_specifics.rst

```diff
@@ -70,7 +72,7 @@ or proxy writer.  In the discovery process, writers are matched with proxy reade
 readers are matched with proxy writers, based on the topic and type names and the QoS
 settings.
 
-Proxies have the same natural hierarchy that ‘normal’ DDSI entities have: each proxy
+Proxies have the same natural hierarchy that "normal" DDSI entities have: each proxy
```

Comment:
This looks like a global replace gone wrong

Reply:
Ah yup, my editor was highlighting unicode apostrophes it didn't like so I global replaced them, but apparently the editor tried to be smart and also replaced the correct ones. I think I've fixed all of them back to what they were.